mico-core/src/main/java/io/github/ust/mico/core/dto/MicoServiceInterfaceDTO.java

```diff
@@ -37,8 +41,15 @@
 public class MicoServiceInterfaceDTO {
```

Comment:
Inconsistent naming. To be consistent with the others it should be *MicoServiceInterfaceStatusDTO*